languages/java/oso/src/test/kotlin/com/osohq/oso/OsoKotlinTest.kt

```diff
@@ -0,0 +1,123 @@
+package com.osohq.oso
+
+import com.osohq.oso.Exceptions.OsoException
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.util.Set
+
+class OsoKotlinTest {
```

Comment:
(Mostly) output of Intellij "convert to kotlin" of the existing `OsoTest`. The use `data class...` instead of `class`